Core/Lib/ExtendedController/WidgetItemAutocomplete.php

```diff
@@ -117,7 +117,7 @@ public function getEditHTML($value)
             $html .= '<span class="input-group-prepend"><span class="input-group-text">'
                 . '<i class="fa fa-keyboard-o" aria-hidden="true"></i></span></span>';
         } else {
-            $html .= '<span class="input-group-btn">'
+            $html .= '<span class="input-group-prepend">'
```

Comment:
This css changes, probably should go in another different PR, but is OK go to merge, but try next time split per issue ;)

Reply:
This was the real begin of this PR.